packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -323,6 +323,11 @@ export interface IGarbageCollectionRuntime {
 
 /** Defines the contract for the garbage collector. */
 export interface IGarbageCollector {
+	/** Tells the time at which session expiry timer started in a previous container.
+	 * This is only set when loading from a stashed container and will be equal to the time at
+	 * which the original container started the session expiry timer.
```

Comment:
Suggestion to mention that it's based on the original container's local client time

```suggestion
	 * This is only set when loading from a stashed container and will be equal to the
	 * original container's local client time when it was loaded (and started the session expiry timer).
```